content/blog/eu_digital_sovereignty.md

```diff
@@ -0,0 +1,83 @@
+
+## What Are US Companies Putting in Place to Reassure Europe of Their Sovereignty?
+
+The biggest US companies providing cloud services in Europe have come forward with their own "sovereign" solutions to try and reassure Europeans that their data is safe and remains under their control.
+
+### Microsoft
+
+Microsoft has launched sovereign solutions which will ensure that all European data stays in Europe and is controlled by Microsoft employees in Europe. It will also include an encryption controller operated by the customer, meaning the cloud provider will not have the decryption key.
+
+### AWS
+
+AWS is about to launch its version of a dedicated cloud for Europe, with infrastructure and personnel from Europe having access to and control of the data stored in Europe.
+
+### Oracle
+
+Oracle is also launching an EU Sovereign Cloud, claiming it will be a completely separate entity from Oracle US. All data will be stored and controlled in Europe.
+
+All these initiatives show that US companies are attempting to demonstrate a genuine move toward European clouds that are independent of any external control.
+
+## The Legal Reality: Jurisdiction Beats Geography
+
+All these initiatives share one fundamental flaw: under the CLOUD Act, geography does not matter—jurisdiction does.
+
+No matter where the data resides, as long as it is stored on hardware owned by an American company or one of its subsidiaries, that data falls under U.S. jurisdiction. If the U.S. government requests access, the company is legally bound to comply, or face severe consequences such as heavy fines, loss of federal contracts, or even exclusion from doing business in the United States.
+
+Even Microsoft admitted before the French Senate that it could not guarantee that, if the U.S. government demanded access to data belonging to French citizens and stored in France, it would be able to block that request.
+
+Microsoft claims that such a request would need to be "legitimate." But how much trust can Europe really place in a company whose ultimate goal is to maximize profit, and whose current leadership, while perhaps law-abiding today, could one day change?
+
+What happens if the United States drifts toward a more authoritarian state, and someone less concerned with legal boundaries takes control of Microsoft? Who will then guarantee that European data remains protected and only accessed under legitimate circumstances?
+
+## When Even Hardware Isn't Safe
+
+The sovereignty problem does not stop at "Who has access to my data?" but also relies on the security of the hardware on which it is stored.
+
+A recent study by KU Leuven and the University of Birmingham has shown a critical flaw in Intel's and AMD's processor chips, allowing any person with physical access to the server to steal data in an almost invisible way.
```

Comment:
Add a small example of how this would play out. e.g. if you host EU data on a US datacenter.
I'm also missing how confidential compute could play into this, e.g. AWS NITRO instances.